### PR TITLE
Cleanup: Pass unicode strings to multistring

### PR DESCRIPTION
**Pull request by nijel**
See https://github.com/translate/translate/issues/3681

### CODE REVIEW

File: translate/storage/poxliff.py [hunk 1]
```diff
@@ -150,8 +151,9 @@ def settarget(self, text, lang='xx', append=False):
 
     def gettarget(self):
         if self.hasplural():
-            strings = [unit.target for unit in self.units]
+            strings = [data.forceunicode(unit.target) for unit in self.units]
             if strings:
+                print(strings)
```
Comment (unho) [inline translate/storage/poxliff.py]: Nit: unnecessary print.

**Comment by unho** [comment]
lgtm after nit

File: translate/storage/poxliff.py (hunk 1 above)
Comment (nijel) [reply translate/storage/poxliff.py]: Apparently I forgot some of the debugging code...

**Comment by nijel** [comment]
Thanks, merged!

**Pull request merged**